File: namsor/csharp/src/OSEG.NamsorExamples/UsRaceEthnicityZIP5Example.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using App.Namsor.Api;
using App.Namsor.Client;
using App.Namsor.Model;

namespace OSEG.NamsorExamples;

public class UsRaceEthnicityZIP5Example
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["api_key"] = "YOUR_API_KEY"};

        try
        {
            var response = new PersonalApi(config).UsRaceEthnicityZIP5(
                firstName: "Keith",
                lastName: "Haring",
                zip5Code: "12345"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling PersonalApi#UsRaceEthnicityZIP5: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: namsor/typescript-axios/src/UsRaceEthnicityZIP5Example.ts
import axios, { AxiosError } from "axios";
import * as api from "namsor_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.PersonalApi(configuration).usRaceEthnicityZIP5(
  "Keith", // firstName
  "Haring", // lastName
  "12345", // zip5Code
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling PersonalApi#usRaceEthnicityZIP5:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
